client_plugin/utils/refcount/refcnt.go

```diff
@@ -428,6 +430,16 @@ func (r *RefCountsMap) syncMountsWithRefCounters(d drivers.VolumeDriver) {
 	defer r.mtx.Unlock()
 
 	for vol, cnt := range r.refMap {
+		// If cnt is nil then this volume entry under mount root can be
+		// purged and the volume detached (if it is attached).
+		if cnt == nil {
```

Comment:
CI run failed as tthe plugin hasn't yet detached the volume when the test code verified the detached state. BUt the plugin logs show that the volume was detached.